manila/share/drivers/netapp/dataontap/client/client_cmode.py

```diff
@@ -2637,9 +2637,16 @@ def set_volume_snapdir_access(self, volume_name, hide_snapdir):
                 'failure-list') or netapp_api.NaElement('none')
             errors = failure_list.get_children()
             if errors:
-                raise netapp_api.NaApiError(
-                    errors[0].get_child_content('error-code'),
-                    errors[0].get_child_content('error-message'))
+                error_code = errors[0].get_child_content('error-code')
+                if error_code == netapp_api.EVOLUMEBUSY:
+                    msg = ('Could not set snapdir access of volume %(volume)s'
+                           ' due to volume busy')
+                    msg_args = {'volume': volume_name}
+                    LOG.warning(msg, msg_args)
+                else:
+                    raise netapp_api.NaApiError(
+                        error_code,
+                        errors[0].get_child_content('error-message'))
```

Comment:
It is about shares that are created from snapshots.
In NetApp this results in creating a volume clone from the parent share's snapshot followed by clone split to make the new volume an independent share. The latter is the part where the volume is `busy`.

The user can mount the share already and would see the .snapshot directory.

Probably it makes sense to implement some re-tries (instead of waiting longer for ensure to do the re-try), because this `busy` situation does not happen on every create share from snapshot operation.